### ISSUE TITLE
.env file loaded twice

### ISSUE THREAD

**Issue by psprint** [zpm-zsh/autoenv#10]
Hello,

```zsh
% cat ~/.env
echo "HOME .env"
% source ~/.zplugin/plugins/horosgrisa---autoenv/autoenv.plugin.zsh
HOME .env
HOME .env
```

**Comment by grigorii-horos** [comment]
@psprint Please, check now

**Comment by Nagasaki45** [comment]
To help keep things organized, the change that @horosgrisa applied and asked @psprint to check is 1194d15f9ff616b7b845dda259fd79973adf7b03.

I would like to add that although I don't fully understand the code I commented out these lines to solve another problem I encountered. I'm using [terminator](https://code.google.com/archive/p/jessies/wikis/Terminator.wiki), and the following cuase the shell prompt to stop working propetly (no color, no git integration etc.):

1. Enter a folder with `.env` file.
2. Split the terminator window into two (`pwd` in the new window will point to the same location, but the environment won't be activated, which is fine).
3. In the new window, activate the environment with `source .env`.
4. Exit the directory.

It may be related to the original issue, or not. In any case, it can indicate that there is an issue with this part of the plugin.
